docs/developers/bitesize/ci/CICD.md

```diff
@@ -0,0 +1,29 @@
+Cromwell uses [Continuous Integration](https://en.wikipedia.org/wiki/Continuous_integration) (CI) testing, along with [Continuous Delivery](https://en.wikipedia.org/wiki/Continuous_delivery) (CD) to the Cromwell-as-a-Service (CaaS) `DEV` environment. [Continuous Deployment](https://en.wikipedia.org/wiki/Continuous_deployment) is not implemented.
```

Comment:
per the Wikipedia link
```suggestion
Cromwell uses [Continuous Integration](https://en.wikipedia.org/wiki/Continuous_integration) (CI) testing, along with [Continuous Delivery](https://en.wikipedia.org/wiki/Continuous_delivery) (CDE) to the Cromwell-as-a-Service (CaaS) `DEV` environment. [Continuous Deployment](https://en.wikipedia.org/wiki/Continuous_deployment) is not implemented.
```

Reply:
CD is used throughout the wikipedia text 32 times. CDE only appears once. I don't even know why CDE is mentioned there. 🤔

Reply:
I assumed to disambiguate from CD as an abbreviation for Continuous Deployment?

Reply:
Although the CD overload is confusing I think it only adds to the confusion to use a term that isn't used by anyone else, yet

Reply:
Since it's only used once more in this document, could we just avoid the contentious acronym altogether and make it say `## Continuous Delivery to CaaS DEV` below?

Reply:
The name of the documentation section is "CI/CD". Embrace the chaos.